FacuReartes/react-laba-final-project-team-2
# SPT2-147 - Home Page server side prefetching

# Description


## Changes

- Updated the `Home Page` to prefetch filter data, handle URL params and filter initial products in server side.
- Mobile responsiveness fix.

### Jira

- [SPT2-147](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-147?atlOrigin=eyJpIjoiNTA3YzY5YTk1NjNhNDkzZjlkN2U0YmQ3ZTEzM2E2MjMiLCJwIjoiaiJ9)
- [SPT2-148](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-148?atlOrigin=eyJpIjoiYTMyNjVmMTRlN2RmNDMyMTgwMzYyMDZhMjk4NzI2NmQiLCJwIjoiaiJ9)

### How to Test

1. Navigate to ['http://localhost:3000/'](http://localhost:3000/).
2. Verify in Network tab in devtools to see no initial fetching for home page products.
3. Change the URL to see initial filter from URL params.

### Checklist



[SPT2-147]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SPT2-148]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-148?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
